plugins/inputs/logstash/README.md

```diff
@@ -0,0 +1,86 @@
+# Logstash Input Plugin
+
+This plugin reads metrics exposed by [Logstash Monitoring API](https://www.elastic.co/guide/en/logstash/current/monitoring-logstash.html).
+
+### Configuration:
+
+```toml
+  ## This plugin reads metrics exposed by Logstash Monitoring API.
+  # https://www.elastic.co/guide/en/logstash/current/monitoring.html
+  #
+  # url = "http://localhost:9600"
+  url = "http://localhost:9600"
+```
+
+### Measurements & Fields:
+
+- logstash_jvm
+    - threads_peak_count
+    - mem_pools_survivor_peak_max_in_bytes
+    - mem_pools_survivor_max_in_bytes
+    - mem_pools_old_peak_used_in_bytes
+    - mem_pools_young_used_in_bytes
+    - mem_non_heap_committed_in_bytes
+    - threads_count
+    - mem_pools_old_committed_in_bytes
+    - mem_pools_young_peak_max_in_bytes
+    - mem_heap_used_percent
+    - gc_collectors_young_collection_time_in_millis
+    - mem_pools_survivor_peak_used_in_bytes
+    - mem_pools_young_committed_in_bytes
+    - gc_collectors_old_collection_time_in_millis
+    - gc_collectors_old_collection_count
+    - mem_pools_survivor_used_in_bytes
+    - mem_pools_old_used_in_bytes
+    - mem_pools_young_max_in_bytes
+    - mem_heap_max_in_bytes
+    - mem_non_heap_used_in_bytes
+    - mem_pools_survivor_committed_in_bytes
+    - mem_pools_old_max_in_bytes
+    - mem_heap_committed_in_bytes
+    - mem_pools_old_peak_max_in_bytes
+    - mem_pools_young_peak_used_in_bytes
+    - mem_heap_used_in_bytes
+    - gc_collectors_young_collection_count
+    - uptime_in_millis
+
+- logstash_process
+    - open_file_descriptors
+    - cpu_load_average_1m
+    - cpu_load_average_5m
+    - cpu_load_average_15m
+    - cpu_total_in_millis
+    - cpu_percent
+    - peak_open_file_descriptors
+    - max_file_descriptors
+    - mem_total_virtual_in_bytes
+    - mem_total_virtual_in_bytes
+
+- logstash_events
+    - queue_push_duration_in_millis
+    - duration_in_millis
+    - in
+    - filtered
+    - out
+
+- logstash_plugins
+  There are 3 categories, input, filter, output. For each will be separate measurement consisting:
+      - tags
+        - type (input|filter|output)
+        - plugin (name of the plugin, eg:beats, stdout)
+      - fields
+        - queue_push_duration_in_millis (for input plugins only)
+        - duration_in_millis
+        - in
+        - out
+
+### Example Output:
+
+```
+$ ./telegraf -config telegraf.conf -input-filter logstash -test
```

Comment:
We're condensing output so if you wouldn't mind removing the command and leading `>`s.